curvefs/src/client/rpcclient/metaserver_client.cpp

```diff
@@ -107,6 +107,15 @@ class MetaServerClientRpcDoneBase : public google::protobuf::Closure {
     MetaServerClientMetric *metric_;
 };
 
+Time* MetaServerClientImpl::CreateTime() {
+    struct timespec now;
+    clock_gettime(CLOCK_REALTIME, &now);
+    Time* tm = new Time();
+    tm->set_sec(now.tv_sec);
+    tm->set_nsec(now.tv_nsec);
+    return tm;
+}
```

Comment:
It would be more appropriate to do something like this,

```cpp
void SetCreateTime(Time* tm) {
    struct timespec now;
    clock_gettime(CLOCK_REALTIME, &now);
    tm->set_sec(now.tv_sec);
    tm->set_nsec(now.tv_nsec);
}

// caller
SetCreateTime(request.mutable_create());
```

Reply:
ok